### PR TITLE
Add trainers list

### PR DESCRIPTION
**Pull request by indefinities**
- still need edit and create implementation

### CODE REVIEW

File: pokedex/src/components/Trainers/AddTrainersModal.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+const { useState, useEffect } = React;
+// TODO: Fix evolve and devolve later
```
Comment (CookieComputing) [inline pokedex/src/components/Trainers/AddTrainersModal.js:6]: is this from pokemon info? maybe remove later haha

Comment (amywhying) [inline pokedex/src/components/Trainers/AddTrainersModal.js:6]: delete this XD

File: pokedex/src/components/Trainers/AddTrainersModal.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        first_name,
+        last_name,
+        username
+
+    }
+
+    return <>
+        <Modal show={show} onHide={handleClose}>
+            <Modal.Header closeButton>
+                <Modal.Title>Add a Pokemon Info!</Modal.Title>
```
Comment (CookieComputing) [inline pokedex/src/components/Trainers/AddTrainersModal.js:23]: rename to Trainer!

File: pokedex/src/components/Trainers/DelTrainersModal.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+export default function DelModal({ show, handleClose, handleDel, national_num }) {
+    let payload = {
+        national_num: parseInt(national_num),
+    }
+
+    return <>
+        <Modal show={show} onHide={handleClose}>
+            <Modal.Header closeButton>
+                <Modal.Title>Delete a Pokemon Info</Modal.Title>
```
Comment (amywhying) [inline pokedex/src/components/Trainers/DelTrainersModal.js:14]: Change this so it's about trainer

File: pokedex/src/components/Trainers/DelTrainersModal.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+
+export default function DelModal({ show, handleClose, handleDel, national_num }) {
```
Comment (CookieComputing) [inline pokedex/src/components/Trainers/DelTrainersModal.js:6]: national_num should be renamed to pk or trainer id, whatever the trainer primary key is

File: pokedex/src/components/Trainers/DelTrainersModal.js (hunk 3 above)
Comment (CookieComputing) [inline pokedex/src/components/Trainers/DelTrainersModal.js:14]: delete a trainer

File: pokedex/src/components/Trainers/EditTrainersModal.js [hunk 5]
```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+const { useState, useEffect } = React;
+// TODO: Fix devolve and evolve later
+
```
Comment (CookieComputing) [inline pokedex/src/components/Trainers/EditTrainersModal.js:7]: same comments for pokemon info and what not

File: pokedex/src/components/Trainers/DelTrainersModal.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    let payload = {
+        national_num: parseInt(national_num),
```
Comment (amywhying) [inline pokedex/src/components/Trainers/DelTrainersModal.js:8]: this should be your trainer's ID

**Review by CookieComputing — APPROVED** [review]
apart from the pokemon info changes, lgtm

File: pokedex/src/components/Trainers/EditTrainersModal.js [hunk 7]
```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+const { useState, useEffect } = React;
+// TODO: Fix devolve and evolve later
```
Comment (amywhying) [inline pokedex/src/components/Trainers/EditTrainersModal.js:6]: delete this

File: pokedex/src/components/Trainers/EditTrainersModal.js [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+
+export default function EditModal({ show, handleClose, pokemonInfo, handleSave, pokemonInfoIndex }) {
+    const [nationalNumber, setNationalNumber] = useState("")
```
Comment (amywhying) [inline pokedex/src/components/Trainers/EditTrainersModal.js:9]: trainer ID

File: pokedex/src/components/Trainers/EditTrainersModal.js [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+    const [name, setName] = useState("")
+    const [photoUrl, setPhotoUrl] = useState("")
+    const [description, setDescription] = useState("")
+    // const [devolvedState, setDevolvedState] = useState("")
```
Comment (amywhying) [inline pokedex/src/components/Trainers/EditTrainersModal.js:13]: these three should be what the trainer has - see here https://lucid.app/lucidchart/f98bafbe-6c22-476e-a849-04b3d1db9f31/edit?invitationId=inv_1a1d07ae-4490-42e8-9900-0ca6f5c7bbbb&page=0_0#

**Pull request merged**